webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -334,3 +348,23 @@ def dimwise_max(vectors: Sequence[T]) -> T:
         return vectors[0]
     else:
         return cast(T, tuple(map(max, *vectors)))
+
+
+C = TypeVar("C", bound=Type)
+
+
+def _recursive_subclasses(cls: C) -> List[C]:
+    "Return all subclasses (and their subclasses, etc.)."
+    # Source: http://stackoverflow.com/a/3862957/1221924
+    return cls.__subclasses__() + [
+        g for s in cls.__subclasses__() for g in _recursive_subclasses(s)
+    ]
+
+
+def get_all_pims_handlers() -> Iterable[
```

Comment:
Are the returned pims handlers used only for listing the extensions? Maybe return the supported extensions here directly and make this function private?